main.pm

```diff
@@ -834,15 +834,17 @@ else {
         load_zdup_tests();
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
+        loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ADDONS")) {
+            loadtest "installation/addon_products_yast2.pm";
+        }
         if (get_var("RT_TESTS")) {
             set_var('INSTALLONLY', 1);
```

Comment:
RT is a flavour of SLE, so I would like it to be handled the same way as the other flavours

See: `sub is_server` and `sub is_jeos` and similar in lines 92-106

Then I would stop using the RT_TESTS variable, and instead use `if is_rt()`

I would also not do install only.. why don't we just have some scenarios that install RT, run the console tests, and run the RT tests? am I misunderstanding something?